README.rst

```diff
@@ -86,14 +86,12 @@ Dependencies
 
 PennyLane-Forest requires the following libraries be installed:
 
-* `Python <http://python.org/>`__ >=3.6
+* `Python <http://python.org/>`__ >=3.7
 
 as well as the following Python packages:
 
 * `PennyLane <http://pennylane.readthedocs.io/>`__
```

Comment:
We may need to add a lower bound on the PennyLane version. Will look more into it.